### DISCUSSION TITLE
Mixing Fluid Requester Station Sizes

### DISCUSSION THREAD

**Discussion by steve-anthony-lee** [Opinionated-Blueprints/10-Books-Full-of-Rails#122] [General]
So if I have a Provider Station for Oil that accepts 1-2-0 fluid wagons. But I also want to have a smaller Sulfuric Acid station that's just 1-1-0, will they mesh well together? I just plopped them down and noticed the  1-1-0 was going to the oil station and then sitting put because the station algorithm was waiting for the 2nd non-existent fluid wagon to fill up.

Should I just put the Sulfuric Acid station and it's train on a separate signal group so they don't fuck with each other? Or just make the SA station a 1-2-0?

**Comment by AspectLL** [comment]
As long as you put the different stations and trains into their own groups it shouldn't be a problem. your 1-1-0 trains should have a separate group that only goes to stations that are it's size and marked with the same group.
